<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpChatWidget</class>
 <widget class="QWidget" name="TcpChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>443</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>13000000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TcpChatWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="chatTextEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="messageLineEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userNameLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="startServerButton">
         <property name="text">
          <string>启动服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="ipTextEdit">
       <property name="maximumSize">
        <size>
         <width>13000000</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>服务器IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="severIpLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>输入端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
